x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/transform/transforms/TransformConfigTests.java

```diff
@@ -187,10 +231,7 @@ public void testDefaultMatchAll() throws IOException {
     }
 
     public void testConstructor_NoFunctionProvided() throws IOException {
-        String json = "{"
-            + " \"source\": {\"index\": \"src\"},"
-            + " \"dest\": {\"index\": \"dest\"}"
-            + "}";
+        String json = "{" + " \"source\": {\"index\": \"src\"}," + " \"dest\": {\"index\": \"dest\"}" + "}";
```

Comment:
Since the formatter decided it fits in one line, could you remove `+` operators?

Reply:
:facepalm: I will look into making the formatter to not do this